wiki/People/Community_Contributors/en.md

```diff
@@ -213,3 +223,13 @@ Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni), which are previou
 | ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Outstanding contribution to the osu! tournament scene and the World Cups |
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Outstanding contribution to the osu! tournament scene and the World Cups |
 | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Outstanding contribution to the osu! IRL events and tournament scene |
+
+## History
+
+During 2018, the old contributor badge was redesigned due to not matching the new website's colour scheme.
+
+![1st design change comparison](img/design-change-1.png)
+
+The design was then changed once again on 21 July 2023, this time split into two versions of the old design, where older contributors (with a user ID below 4,000,000) saw a revert to the original while newer contributors received a more colourful variant.
```

Comment:
```suggestion
The design was changed again on 21 July 2023, this time split into two versions of the initial design, where older contributors (with a user ID below 4,000,000) saw a revert to the original while newer contributors received a more colourful variant.
```
very nitpick but writing "then" kinda makes it sound like this was a related event happening in succession
and then old->initial since "old" could at this point refer to the result of first change